packages/experiments/src/components/Announced/Announced.types.ts

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { AnnouncedBase } from './Announced.base';
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface IAnnounced {}
+
+export interface IAnnouncedProps extends React.Props<AnnouncedBase> {
+  /** Theme from the Higher Order Component */
+  theme?: ITheme;
+
+  /** Call to provide customized styling that will layer on top of the variant rules. */
+  styles?: IStyleFunctionOrObject<IAnnouncedStyleProps, IAnnouncedStyles>;
+
+  /**
+   * Optional callback to access the IAnnounced interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IAnnounced) => void;
+
+  /**
+   * The status message provided as screen reader output
+   */
+  message?: string;
```

Comment:
Would it ever be useful for the `message` to be invoked as a function which returns a `string` from a developer standpoint?